SPEC.md

```diff
@@ -496,21 +748,47 @@ hex := sign? '0x' hex-digit (hex-digit | '_')*
 octal := sign? '0o' [0-7] [0-7_]*
 binary := sign? '0b' ('0' | '1') ('0' | '1' | '_')*
 
-boolean := 'true' | 'false'
+keyword := boolean | '#null'
 
-escline := '\\' ws* (single-line-comment | newline)
+boolean := '#true' | '#false'
 
-linespace := newline | ws | single-line-comment
+escline := '\\' ws* (single-line-comment | newline | eof)
 
 newline := See Table (All line-break white_space)
 
-ws := bom | unicode-space | multi-line-comment
+ws := unicode-space | multi-line-comment
 
 bom := '\u{FEFF}'
 
-unicode-space := See Table (All White_Space unicode characters which are not `newline`)
+disallowed-literal-code-points := See Table (Disallowed Literal Code Points)
+
+equals-sign := See Table ([Equals Sign](#equals-sign))
```

Comment:
Not the biggest issue but this rule is duplicated now, the other one is under the prop/value/type rules.